### DISCUSSION TITLE
Requesting Guidance on Modifying Code to Send Signals to the Buzzer Port of Crazyflie 2.0's Big Quad Deck

### DISCUSSION THREAD

**Discussion by NarekHarutyu** [bitcraze/discussions#777] [Q&A]
I am seeking assistance in understanding the process of sending signals, specifically adjusting the frequency, to the buzzer port of the Big Quad deck attached to the Crazyflie 2.0. Could you kindly provide guidance on the specific section of the code that needs modification to achieve this?

**Comment by ataffanel** [comment]
The buzzer connector is connected to the Crazyflie's IO_4 pin which is connected to the STM32405 PC12 pin (according to https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#expansion-port-pinout). This pin is not connected to any timer output in the STM32 so it is not easily possible to output a frequency on this pin.

This connector was designed to connect to an active buzzer that would emit sound when powered ON.
